exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/TraceRequestMarshaler.java

```diff
@@ -27,58 +27,61 @@
 import java.util.List;
 import java.util.Map;
 
-final class TraceMarshaler {
+/** {@link Marshaler} to convert SDK {@link SpanData} to OTLP ExportTraceServiceRequest. */
+public final class TraceRequestMarshaler extends MarshalerWithSize implements Marshaler {
```

Comment:
internal boilerplate here.

Reply:
I noticed we have a lot of public classes without the boilerplate, both in this package and others like context / api. Wonder if it's a sign of being a fruitless effort :P

Reply:
I try to catch them when I remember, but yeah, we're definitely still missing some.